Templates/DefaultProject/Template/Platform/Android/android_project.json

```diff
@@ -1,7 +1,7 @@
 {
     "Tags": ["Android"],
     "android_settings" : {
-        "package_name" : "com.lumberyard.${Name}",
+        "package_name" : "com.o3de.${Name}",
```

Comment:
Is it really correct to use the "o3de" naming scheme in the Project Templates?
Seems like this should probably not be set

Reply:
The Android Gradle project generation unfortunately requires this here

Reply:
yeah, maybe that could be an additional argument to the android project generator and that this file is used as a template instead

Reply:
That would certainly work, may be on the edge of this change's scope though.

In an ideal world, the Android project builder files would be included in the template instead and things like this would go away but that requires even more and significant changes.

Reply:
Who should decide what this should be?

Reply:
I'd say the the change to "org.o3de.${Name}" is fine for now, anything beyond that will require some careful consideration.

Reply:
I am find with it being `"org.o3de.${Name}"` for now